internal/audit/kafka/conf.go

```diff
@@ -56,6 +75,10 @@ func (c *Conf) SetDefaults() {
 	c.ClientID = defaultClientID
 	c.MaxBufferedRecords = defaultMaxBufferedRecords
 	c.Compression = []string{CompressionSnappy, CompressionNone}
+
+	if c.Authentication.TLS != nil {
+		c.Authentication.TLS.InsecureSkipVerify = false
+	}
```

Comment:
I don't think this necessary? Won't the newly-created conf always have `c.Authentication.TLS == nil` (and `false` is the default value for booleans anyway)?

```suggestion
```

Reply:
It's not necessary. The goal of this was to make it clear that this is the default. Will remove 👍